pkg/services/oauthtoken/oauth_token_test.go

```diff
@@ -72,6 +72,7 @@ func TestService_HasOAuthEntry(t *testing.T) {
 		},
 	}
 	for _, tc := range testCases {
+		tc := tc
```

Comment:
Is this needed?

Reply:
to silence memory aliasing warning from linters :)